src/components/TeamCard/index.jsx

```diff
@@ -17,7 +17,7 @@ const TeamCard = (props) => {
 							</div>
 							<div className="col-start-9 col-span-2">
 								<button className="w-24 whitespace-nowrap rounded-full border-2 px-3 py-1 text-base md:text-gray-800">
-									{props.round + "st Round"}
+									{"2nd Round"}
```

Comment:
Since this is a reusable component you don't have to change the (hardcode) relevant "round" here. Just parse the correct round where you call this functional component.